apis/apps/v1beta3/emqxplugin_types.go

```diff
@@ -24,9 +24,11 @@ import (
 type EmqxPluginSpec struct {
 	//+kubebuilder:validation:Required
 	PluginName string `json:"pluginName,omitempty"`
```

Comment:
Add link for plugin list: https://www.emqx.io/docs/en/v4.4/advanced/plugins.html#list-of-plugins